tools/archetype/binding/src/main/resources/archetype-resources/src/main/java/handler/__bindingIdCamelCase__Handler.java

```diff
@@ -20,28 +23,28 @@
 /**
  * The {@link ${bindingIdCamelCase}Handler} is responsible for handling commands, which are
  * sent to one of the channels.
- * 
+ *
  * @author ${author} - Initial contribution
  */
 public class ${bindingIdCamelCase}Handler extends BaseThingHandler {
 
-    private Logger logger = LoggerFactory.getLogger(${bindingIdCamelCase}Handler.class);
+    private final Logger logger = LoggerFactory.getLogger(${bindingIdCamelCase}Handler.class);
```

Comment:
Thanks for your time! Your usecase sounds valid, although it is really an exception, lets rephrase my statement ;-)

> In generally it should not be used, because its makes it less clear where the log statements are actually originating from. If we want to guard against copy-paste errors lets follow the manual and use: `MethodHandles.lookup().lookupClass()`

Did you have any other review comments?